submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/stats/web/StatsResource.kt

```diff
@@ -35,17 +38,33 @@ class StatsResource(
     @GetMapping("/{type}", produces = [APPLICATION_JSON_VALUE])
     @ResponseBody
     @ApiOperation("List all the registered submission stats of the given type")
-    @ApiImplicitParam(name = "X-SESSION-TOKEN", value = "Authentication token", required = true, paramType = "header")
+    @ApiImplicitParams(value = [
+        ApiImplicitParam(
+            name = "X-SESSION-TOKEN",
```

Comment:
this one is not needed as it added to every endpoint as part of the configuration